src/components/SprCard.tsx

```diff
@@ -23,17 +24,23 @@ export const SprCard = ({
                 <Stat>
                     <Flex direction='column' align='center'>
                         <SprTag props={{ size: 'xl' }} player={player} />
-                        <StatLabel fontSize='20'>
-                            Season Power Ranking
-                        </StatLabel>
+                        <StatLabel fontSize='20'>SPR</StatLabel>
                         <Tooltip label='Average SPR change from recent games, up to the last five'>
                             <StatHelpText fontSize='14'>
-                                <StatArrow
-                                    type={
-                                        sprTrend > 0 ? 'increase' : 'decrease'
-                                    }
-                                />
-                                {sprTrend}
+                                {sprTrend !== 0 ? (
+                                    <>
+                                        <StatArrow
+                                            type={
+                                                sprTrend > 0
+                                                    ? 'increase'
+                                                    : 'decrease'
+                                            }
+                                        />
+                                        {sprTrend}
+                                    </>
+                                ) : (
+                                    'No Season 1 SPR'
```

Comment:
Can we just have an em-dash here instead, so it matches the other SPR information when not enough data is available? 
This is also more clear, since "No Season 1 SPR" isn't technically correct in some cases.
```suggestion
 {sprTrend !== 0 ? (
                                    <>
                                        <StatArrow
                                            type={
                                                sprTrend > 0
                                                    ? 'increase'
                                                    : 'decrease'
                                            }
                                        />
                                        {sprTrend}
                                    </>
                                ) : (
                                    '—'
                                    )}
                                    </StatHelpText>
                        </Tooltip>
                    </Flex>
```

Reply:
i think we can just make it a dash, my main concern is knowing what the tiny dash is when there is no spr change right now.

Just trying to think of a good way to show this while we are waiting for the season to start.

Reply:
The hover tooltip will still be there. we can also have a "0" and a dash instead of the icon

Reply:
I wish stat arrow would support "no change" instead of only "increase"/"decrease"